models/peep.rb

```diff
@@ -0,0 +1,75 @@
+require 'pg'
+
+class Peep
+
+  attr_reader :message, :creator, :id, :time
+
+  def initialize(message:, creator:, id:, time:)
+    @creator = creator
+    @id = id
+    @message = message
+    @time = format(time)
+  end
+
+  def self.connect_to_db
+    if ENV['RACK_ENV'] == "test"
+      @@connection = PG.connect :dbname => 'peeps_test_manager'
+    else
+      @@connection = PG.connect :dbname => 'peeps_manager'
```

Comment:
Good idea to extract database connection to another method and keep the code dry, you could maybe even extract the database work into another class.

Reply:
Ooh! I like this!